versions/appinfo/0.0.1.yaml

```diff
@@ -0,0 +1,24 @@
+metadata:
+  configVersion: 0.0.2 # this is deliberately wrong for testing
+environments:
+  default: # base configuration that can be overridden
+    android:
+      platform: Android
+      available: true
+      minimumVersion: 1.0.0
+      recommendedVersion: 1.2.0
+      releaseFlags:
+        featureOne: true
+        featureTwo: false
+    ios:
+      platform: iOS
+      available: true
+      minimumVersion: 2.0.1
+      recommendedVersion: 2.2.0
+      releaseFlags:
+        featureOne: true
+        featureTwo: true
+  production:
+    android:
+      releaseFlags:
+        featureOne: false # turn featureOne off in prod for Android
```

Comment:
Personal opinion, but comments in YAML is a bad idea. I've known it break things in unpredictable and hard to diagnose ways.

Reply:
Let's investigate your suggestion to use TOML instead @jflm 👍